examples/di/README.md

```diff
@@ -12,9 +12,92 @@ Data Interpreter is an agent who solves data-related problems through codes. It
 - Tool usage: web page imitation
 - Tool usage: web crawling
 - Tool usage: text2image
-- Tool usage: email summarization and response\
+- Tool usage: email summarization and response
 - More on the way!
 
 Please see the [docs](https://docs.deepwisdom.ai/main/en/guide/use_cases/agent/interpreter/intro.html) for more explanation.
 
-We are continuously releasing codes, stay tuned!
+## Paper Experiments
+
+Before running the experiments, download the [di_dataset](https://drive.google.com/drive/folders/17SpI9WL9kzd260q2DArbXKNcqhidjA7s?usp=sharing) and place it in the specified path (default `DATA_PATH` or any other path you prefer). Note that the `DATA_PATH = METAGPT_ROOT / "data"`.
```

Comment:
in the specified path (default `DATA_PATH`, where `DATA_PATH = METAGPT_ROOT / "data"`).